sdk/base-sys/src/bindings/types.rs

```diff
@@ -4,6 +4,12 @@ use miden_stdlib_sys::{Felt, Word};
 #[derive(Copy, Clone)]
 pub struct AccountId(Felt);
 
+impl AccountId {
+    pub fn as_felt(&self) -> Felt {
```

Comment:
*Nit:*: Since `AccountId` is `Copy`, writing conversions like this as instead:

```rust
#[inline(always)]
pub const fn as_felt(self) -> Felt {
    self.0
```

will basically compile down to a no-op (possibly even without a copy in some cases, if the `AccountId` is not behind a reference/has only a single use).

Not something you need to fix, but could improve codegen in some cases, so mostly just worth keeping in the back of your mind.

Note - the use of `#[inline(always)]` here is because we know that doing so is _always_ strictly better than leaving it unlined. That's not always the case for small functions, but for accessors like this, it is pretty much always worth it IMO.

Reply:
Cool! `no-op`, ha! Thank you.